.github/workflows/main.yml

```diff
@@ -124,7 +124,7 @@ jobs:
         run: |
           pip install "numpy${{ matrix.ver.np }}" "scipy${{ matrix.ver.sp }}"
           python -m pytest --cov gstools --cov-report term-missing -v tests/
-          python -m coveralls --service=github
+          #python -m coveralls --service=github
```

Comment:
Why is this commented out?

Reply:
The coveralls service was down once and stopped the runners completely, but I wanted to check if they complete successfully.
But yeah, that change shouldn't make it into main.